signin.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
```

Comment:
`<html>`태그에는 `lang`속성을 같이 주는게 권장되는데요. 언어코드를 추가해보셔도 좋을거 같아요
```suggestion
<html lang="ko">
```